git_cookbook.md

```diff
@@ -20,3 +20,23 @@ function pr () {
   open https://github.com/$repo/compare/$branch?expand=1
 }
 ```
+
+#### A less verbose status command
+Includes only the files that changed and omits the boilerplate about the branch, etc.  Very handy to alias as `git s`.
+```
+git status -s
+``` 
+
+#### A better overview of history
+This command lists commits one-per-line, in color, with a graph of the history on the left.  Very handy for quickly looking at history or checking the relationships between branches.  I alias it as `git lg`.
+
+```
+git log --oneline --decorate --graph --all -100
```

Comment:
Making a note is fine too.  The description suggests that it is a better view of history, which implies all history.  This is why I wouldn't include it.  Does your history viewer not handle long `--oneline` history well?  I think `less` has handled it fine for me.